src/appwindow.cpp

```diff
@@ -1302,6 +1300,12 @@ void AppWindow::on_actionFullScreen_toggled(bool checked)
     auto state = windowState();
     state.setFlag(Qt::WindowFullScreen, checked);
     setWindowState(state);
+    if (!SettingsHelper::isOnFullScreenDialogShown())
+    {
+        QMessageBox::information(this, tr("How to exit full-screen"),
+                                 tr("Press the F11 key on your computer's keyboard to exit full-screen mode."));
+        SettingsHelper::setOnFullScreenDialogShown(true);
```

Comment:
Should change as you change config file name.

Reply:
Changed